doc/recognition.xml

```diff
@@ -356,8 +356,17 @@ set this to <K>false</K>. The result is either <K>fail</K> or
 <K>false</K> if the method fails or a recognition info record <C>ri</C>.
 If the method created a leaf then <C>ri</C> will be a leaf, otherwise
 it will have the attribute <Ref Attr="Homom"/> set, but no factor or
-kernel have been created or recognised yet. You can use for example
-the methods in <Ref Var="FindHomMethodsPerm"/> or
+kernel have been created or recognised yet.
```

Comment:
I know it's not your sentence, but since we're touching it, could you please update to:
```suggestion
kernel will have been created or recognised yet.
```